src/workerd/api/blob.c++

```diff
@@ -143,13 +143,23 @@ public:
       : unread(blob->data),
         blob(kj::mv(blob)) {}
 
+  // Attempt to read a maximum of maxBytes from the remaining unread content of the blob
```

Comment:
Per our little discussion in person here:

* I'd be willing to switch to comments-before-declaration if the team generally votes for it, but this is a broader discussion we need to have outside this PR.
* If we are going to convert, I really want us to write a tool that converts all the comments at once, rather than do it incrementally (which inevitably would just mean we have no consistent style). Again, willing to yield if the team broadly wants to take the incremental approach.
* An alternative might be to figure out how to support post-declaration comments in clangd, so that vscode tooltips actually show the correct comment. At least some on the team seem to think this would be a better fix.

Reply:
I've backed the comment change out of this PR